cg/cli/set/base.py

```diff
@@ -163,41 +163,27 @@ def list_changeable_sample_attributes(
 
 def show_set_sample_help(sample_obj: models.Sample = "None") -> None:
     """Show help for the set sample command"""
-    LOG.info("sample_id: optional, internal_id of sample to set value on")
-    show_option_help(long_name=OPTION_LONG_SKIP_LIMS, help_text=HELP_SKIP_LIMS)
-    show_option_help(short_name=OPTION_SHORT_YES, long_name=OPTION_LONG_YES, help_text=HELP_YES)
-    show_option_help(
-        short_name=OPTION_SHORT_KEY_VALUE, long_name=OPTION_LONG_KEY_VALUE, help_text=HELP_KEY_VALUE
-    )
-    list_changeable_sample_attributes(sample_obj, skip_attributes=NOT_CHANGEABLE_SAMPLE_ATTRIBUTES)
-    LOG.info(f"To set apptag use '{OPTION_SHORT_KEY_VALUE} application_version [APPTAG]")
-    LOG.info(f"To set customer use '{OPTION_SHORT_KEY_VALUE} customer [CUSTOMER]")
-    LOG.info(
-        f"To set priority use '{OPTION_SHORT_KEY_VALUE} priority [priority as text or " f"number]"
-    )
-
-
-def show_option_help(short_name: str = "", long_name: str = "", help_text: str = "") -> None:
-    """Show help for one option"""
-    help_message = f"Use "
-
-    if short_name:
-        help_message += f"'{short_name}'"
-
-    if short_name and long_name:
-        help_message += " or "
+    LOG.info(f"Below is a set of changeable sample attributes, to combine with -kv flag:\n")
 
-    if long_name:
-        help_message += f"'{long_name}'"
+    list_changeable_sample_attributes(sample_obj, skip_attributes=NOT_CHANGEABLE_SAMPLE_ATTRIBUTES)
 
-    if help_text:
-        help_message += f": {help_text}"
 
-    LOG.info(help_message)
+@set_cmd.command()
+@click.option("-s", "--sample_id", help="List all available modifiable keys for sample")
+@click.pass_obj
+def list_keys(
+    context: CGConfig,
+    sample_id: Optional[str],
+):
+    """List all available modifiable keys"""
+    status_db: Store = context.status_db
+    sample_obj: models.Sample = status_db.sample(internal_id=sample_id)
```

Comment:
```suggestion
    sample: models.Sample = status_db.sample(internal_id=sample_id)
```
We (and your editor) knows is is an object from the type hint, so no need to have it in the variable name

Reply:
I really need to learn these conventions, at the moment I'm mostly trying to learn from seeing how it has been implemented in the established code, and from what I have been able to find it seems to be the convention to write sample_obj in this case. I'm worried that it will be confusing if I deviate from the convention in this function only. But  let me know if you still think it's a good idea

Reply:
To clarify the alternatives seem to be:
1. to change to sample instead of sample_obj in a lot of different areas of cg (including microSALT, mutant, cgstats, store_helpers, workflow analysis, balsamic...etc) 
2. to change it only here (it shouldn't break anything, but maybe be a bit confusing, and gradually we could change the naming elsewhere to be consistent)
3. to keep it as it is

Reply:
I think option nr 2 is a good choice here